openflow_input/bsn_tlv

```diff
@@ -1305,3 +1305,22 @@ struct of_bsn_tlv_lr_all_enabled : of_bsn_tlv {
     uint16_t type == 178;
     uint16_t length;
 };
+
+enum ofp_bsn_port_mode(wire_type=uint16_t) {
+    OFP_BSN_PORT_MODE_4XX = 0,
```

Comment:
Can we add OFP_BSN_PORT_MODE_NONE = 0, which can be used for default assignment. And also it helps if it is default value or controller configured value. Controller should never use OFP_BSN_PORT_MODE_NONE.